### ISSUE TITLE
Reserve fuel tank mass is ignored

### ISSUE THREAD

**Issue by Freaky** [EDCD/coriolis#560]
The reserve fuel tank accounts for up to [1.16t](https://forums.frontier.co.uk/threads/reserve-tank-sizes-for-all-ships.460434/) of ship mass, which Coriolis does not take into account.  This makes it slightly over-estimate practical jump range.

**Comment by Freaky** [comment]
Just to confirm this shouldn't be added to laden mass generally, I built [this Anaconda](https://coriolis.io/outfit/anaconda?code=A0patdF3lndisxf2--------------000i-------------.Iw18ZXEA.Aw18ZXEA.H4sIAAAAAAAAA2P5Z8jAwPCXFUjwV7z5%2F1%2FozL3%2F%2F%2F%2Bz%2FUsAiTKCRCX%2BAEUNfvz%2Fr8fzDyj1n4EBAOHdkYw0AAAA.EweloBjEoUwQwOYBtYhARgtmuJA%3D&bn=Marginal%20Thrust), which is right flat up against the 473t limit of a 4D thruster: `"UnladenMass":469.000000`, so sayeth Elite.

Coriolis-data seems to think it.s 472.5t, and only rounded up for display, but this appears to be incorrect - this abomination flies ~~just fine~~.

**Comment by Noobilite** [comment]
Does reserve Fuel count towards the mass limit of engines and other things in game or not. I've never figured this out.

**Comment by Freaky** [comment]
No, I've built this Anaconda and [a Type-9](https://coriolis.io/outfit/type_9_heavy?code=A0p6tjFflnd8sif3---------3f-------3w3xf1.Iw18ZQ%3D%3D.Aw18ZQ%3D%3D.H4sIAAAAAAAAA2P4x87AwPCXBUhwGzAxMPD2cDEwCD5gZGAQOsLJwKAyAcj6z%2FRPHKQIyGT4Uwwk%2BHdwA%2BW%2FAPnKEn%2F%2B%2F%2F%2FP%2FE8KJM8KklL59f%2B%2FEEhepEaUgUHiDlDw%2F3%2BgBABdrR52aQAAAA%3D%3D.EweloBhBGKoUwIYHMA28QwtkddA%3D&bn=Dead%20Slow) flat up against the thruster mass limit - these builds wouldn't work if the reserve tank mass was counted.

**Comment by felixlinker** [comment]
Excellent report, thanks!

**Comment by felixlinker** [comment]
Does your commit fix the issue? And if yes, could you rebase your changes onto develop and create a new PR?

**Comment by Freaky** [comment]
It's a partial fix which depends on https://github.com/EDCD/coriolis-data/pull/79 - though it has fallback so isn't blocked by it.  I've confirmed the jump ranges are correct across multiple ships.

I rebased and re-pushed, shouldn't need a new PR?
